core/templates/dev/head/pages/library/Library.js

```diff
@@ -28,16 +28,16 @@ oppia.constant('LIBRARY_PAGE_MODES', {
 
 oppia.controller('Library', [
   '$scope', '$http', '$modal', '$rootScope', '$window', '$timeout',
-  'i18nIdService', 'UrlService', 'ALL_CATEGORIES', 'SearchService',
-  'WindowDimensionsService', 'UrlInterpolationService', 'LIBRARY_PAGE_MODES',
-  'LIBRARY_TILE_WIDTH_PX', 'AlertsService',
+  'ConstructTranslationIdsService', 'UrlService', 'ALL_CATEGORIES',
+  'SearchService', 'WindowDimensionsService', 'UrlInterpolationService', 
```

Comment:
Remove the extra space at the end of line 32.

Reply:
Done